Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -1,10 +1,77 @@
 import SwiftUI
 import UIKit
 
-public class SimplifiedLoginViewController: UIViewController {
+public struct SimplifiedLoginViewModel {
+    var localisation: Localisation
+    var iconNames: [String]
+    let schibstedLogoName = "sch-logo"
+    
+    let displayName = "Daniel.User" // TODO: Need to be fetched
+    let clientName = "Finn" // TODO: Need to be fetched
+    
+    init?(env: ClientConfiguration.Environment){
+        let resourceName: String
+        let orderedIconNames: [String]
+        switch env {
+        case .proCom:
+            resourceName = "for_use_simplified-widget_translations_sv"
+            orderedIconNames = ["Blocket", "Aftonbladet", "SVD", "Omni", "TvNu"]
+        case .proNo:
+            resourceName = "for_use_simplified-widget_translations_nb"
+            orderedIconNames = ["Finn", "VG", "Aftenposten", "E24", "BergensTidene"]
+        case .proFi:
+            resourceName = "for_use_simplified-widget_translations_fi"
+            orderedIconNames = ["Tori", "Oikotie", "Hintaopas", "Lendo", "Rakentaja"]
+        case .proDk:
+            resourceName = "for_use_simplified-widget_translations_da"
+            orderedIconNames = ["Tori", "Oikotie", "Hintaopas", "Lendo", "Rakentaja"] //TODO: NEED DK 5 brands with icons
+        case .pre:
+            resourceName = "simplified-widget_translations_en"
+            orderedIconNames = ["Blocket", "Aftonbladet", "SVD", "Omni", "TvNu"] // Using SV icons
+        }
+        
+        let decoder = JSONDecoder()
+        guard
```

Comment:
I prefer to have the first condition inline with guard word. We need to add swift-lint or swift-format :P